### ISSUE TITLE
SSH gateway stops the workspace immidiately upon disconnect after (plan inactivity timeout) amount of useage before disconnect

### ISSUE THREAD

**Issue by axonasif** [gitpod-io/gitpod#12209]
### Bug description

excerpt from an email:
_Hi, I am having a very obnoxious issue with my gitpod containers shutting down early. I am using the “Command Line (SSH)” method of connecting to my pods, so that I can do development using tmux+vim. The experience works well until my SSH connection disconnects. Intuitively, this should start the timeout period on the container and it should automatically shut down 30 minutes later. Unfortunately, the observed behavior is that the timeout period is never reset and so if the container has been alive for longer than 30 minutes it immediately shuts down upon disconnect. This means if I shut my laptop to walk to another room, my gitpod container is shut down and it takes me several minutes to restart the container and open all of the files that I had opened._

### Steps to reproduce

1. Create a workspace and connect via ssh.
2. Work on a file in the project directory or do anything to keep it alive **for a period of 30+ minutes.** (personal plan)
3. Confirm that the gitpod dashboard timer for that specific workspace reports more than 30 minutes of usage.
4. Disconnect the SSH session gracefully.
5. Observe that the workspace immediately stopped. 

### Expected behavior

It should not immediately stop when the workspace was used for more time than `{plan inactivity timeout}`

### Anything else?

cc @CGamesPlay

**Comment by CGamesPlay** [comment]
Yeah, that about sums it up. I will say that "immediately" might be a minute or two. But much less than 30 minutes. When reporting this bug I waited 5 minutes and saw it had been shut off.

**Comment by akosyakov** [comment]
Are you aware of close timeouts: 

From https://www.gitpod.io/docs/life-of-workspace#timeouts:
> For convenience, closing the browser window/tab containing the workspace reduces the timeout to 3 minutes.

This statement is out dated, we extended close timeout to 5 mins. But it applies to ssh terminal sessions as well. If you don't provide any input within 5 minutes after disconnection we consider that you abandoned a session and stop a workspace to reduce costs on your side. Does it correlate with your experience? In our tracing I see close signals coming from one of your workspaces.

**Comment by CGamesPlay** [comment]
That seems to corrrelate pretty well with my experience. But it really makes using the SSH connection inconvenient! When I am developing on my iPad, locking the screen will cause a disconnect. If I go to the kitchen to get a drink I’ll have to restart my workspace :’( Is there any workaround for this? Some way to go back to the normal 30 minute timeout?

To be clear I know you will move to usage-based pricing and would want a 30 minute timeout once I’m paying by the second. But I don’t want a 5 minute timeout, because it’s too short.
